homeassistant/components/miele/strings.json

```diff
@@ -194,9 +194,9 @@
       "program_phase": {
         "name": "Program phase",
         "state": {
-          "2nd_espresso": "2nd espresso coffee",
-          "2nd_grinding": "2nd grinding",
-          "2nd_pre_brewing": "2nd pre-brewing",
+          "z_2nd_espresso": "2nd espresso coffee",
+          "z_2nd_grinding": "2nd grinding",
+          "z_2nd_pre_brewing": "2nd pre-brewing",
```

Comment:
`z_`?

Reply:
I got problems in MieleEnum (or upstream)  with keys beginning with a numeric character. I can change to "second_espresso" etc in the keys.

Reply:
I would prefer that one yes

Reply:
And, we should make sure this is not a breaking change

Reply:
It is a breaking change. For users with Miele coffee systems that that have created an automation that trigger on "2nd_pre-brewing". The value has been available since 4 days...
My suggestion is that we make a note under "Breaking change", but refrain from spending energy and time on making a smooth migration for the most likely non-existent users.

There are many advantages to use the Enum class for these code tables. One small drawback is that identifiers in python cannot have leading numeric characters.